src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
 		// Find delegate handlers
 		// Black-hole SVG <use> instance trees (#13180)
+		//
+		// Support: Firefox
```

Comment:
This was from http://bugs.jquery.com/ticket/3861 and still around when I rewrote things in 1.8, but seems to have disappeared at some point: http://jsbin.com/qoqofa/1/edit?js,console,output